lib/Sema/TypeChecker.cpp

```diff
@@ -462,6 +462,18 @@ static void typeCheckFunctionsAndExternalDecls(SourceFile &SF, TypeChecker &TC)
       TC.finalizeDecl(decl);
     }
 
+    // Type check conformance contexts.
+    for (unsigned i = 0; i != TC.ConformanceContexts.size(); ++i) {
```

Comment:
Good choice; I like doing this as a later "pass" anyway at this point, because it makes us less likely to be order-sensitive in the interim.